pkg/products/monitoringcommon/alertmanagerConfig_test.go

```diff
@@ -264,24 +264,6 @@ func TestReconciler_reconcileAlertManagerSecrets(t *testing.T) {
 			wantErr: "secret key is undefined in pager duty secret",
 			want:    integreatlyv1alpha1.PhaseFailed,
 		},
-		{
-			name: "fails when dead mans snitch secret cannot be found",
-			serverClient: func() k8sclient.Client {
-				return fakeclient.NewFakeClientWithScheme(basicScheme, smtpSecret, pagerdutySecret, alertmanagerRoute)
-			},
-			wantErr: "could not obtain dead mans snitch credentials secret: secrets \"test-dms\" not found",
-			want:    integreatlyv1alpha1.PhaseFailed,
-		},
-		{
-			name: "fails when dead mans snitch url is not defined",
-			serverClient: func() k8sclient.Client {
-				emptyDMSSecret := dmsSecret.DeepCopy()
-				emptyDMSSecret.Data = map[string][]byte{}
-				return fakeclient.NewFakeClientWithScheme(basicScheme, smtpSecret, pagerdutySecret, emptyDMSSecret, alertmanagerRoute)
-			},
-			wantErr: "url is undefined in dead mans snitch secret",
-			want:    integreatlyv1alpha1.PhaseFailed,
-		},
```

Comment:
With the removal of these test is there any other unit test that could be added?

Reply:
Done. These 2 Tests return back, and modified similar to SMTP test. I think now it's correct. Thank you very much for this point!